.github/workflows/test.yml

```diff
@@ -16,9 +16,7 @@ jobs:
       - name: Install dependencies
         run: npm ci
       - name: Run tests
-        uses: percy/exec-action@v0.3.1
-        with:
-          custom-command: "npm test"
+        run: npx @percy/cli exec -- npm test
```

Comment:
This is close, but it won't work how you might thing. This evaluates to: `npx @percy/cli exec -- percy exec -- testcafe chrome:headless tests` which will create two servers and break. We can just change this line to be `npm test` and it'll work 👍🏼

Reply:
Ah, because [npm test is already hooked up to Percy](https://github.com/percy/example-percy-testcafe/blob/7aba2528144be54a1d87979e593f3767e7616087/package.json#L14). Got it.

Reply:
Exactly 😃 and since the CLI is a dependency of the package we could also do `npx percy exec -- testcafe chrome:headless tests` if we wanted to. But using an npm script does almost the same thing as npx (npx will do extra stuff where it goes to the npm registry if the dependency isn’t installed locally).